include/swift/Runtime/Config.h

```diff
@@ -18,7 +18,6 @@
 #define SWIFT_RUNTIME_CONFIG_H
 
 #include "swift/Basic/Compiler.h"
-#include "swift/Runtime/CMakeConfig.h"
```

Comment:
Does this work on Windows too? I think that we rely on this to know about static vs dynamic

Reply:
Welp, seems that the contents are now:
```
// This file is processed by CMake.
// See https://cmake.org/cmake/help/v3.0/command/configure_file.html.

#ifndef SWIFT_RUNTIME_CMAKECONFIG_H
#define SWIFT_RUNTIME_CMAKECONFIG_H

#define SWIFT_VERSION_MAJOR "6"
#define SWIFT_VERSION_MINOR "3"

#endif
```